shared/p2p/service.go

```diff
@@ -231,7 +231,18 @@ func (s *Server) Send(msg proto.Message, peer Peer) {
 	s.Broadcast(msg)
 }
 
-// Broadcast a message to the world.
+// Broadcast publishes a message to all localized peers using gossipsub. 
+// msg must be a proto.Message that can be encoded into a byte array.  
+// It publishes the first 100 chars of msg over the msg's mapped topic. 
+// To map a messageType to a topic, use RegisterTopic.
+//
+// It logs an error if msg is not a protobuf message, 
+// if msg cannot be encoded into a byte array, 
+// or if the server is unable to publish the message over gossipsub. 
+//
+//	 msg := make(chan p2p.Message, 100) // Choose a reasonable buffer size!
```

Comment:
The formatting here is a bit off, can you fix it?

Reply:
Hey - what exactly were you referring to in the formatting?

Reply:
I would only indent the code starting at line 243 with 2 spaces to mach 245 for nicer godocs - is that what you meant @prestonvanloon ?

Reply:
Fixed!